### PR TITLE
Exposed a method to change cached sessionId

### PR DESCRIPTION
**Pull request by detj**
Needed a way to change the already set session id during login and make further requests. This is helpful when a previous sessionId is already available and I don't wan't to create a new session. The previous sessionId is stored too, just in case we need to switch back, though I haven't implemented switching back.

Added a small test too. I'm thinking of implementing a better test with mocha when I have some time.

Let me know if this is helpful.

### CODE REVIEW

File: src/magento.js [hunk 1]
```diff
@@ -182,4 +183,9 @@ Magento.prototype.login = function(callback) {
   return this;
 };
 
+Magento.prototype.changeSession = function(sessId) {
+	this.prevSessId = this.sessionId;
+	this.sessionId = sessId;
```
Comment (tmarshall) [inline src/magento.js]: add a `return this;` at the end of this function, so it's chainable

**Comment by tmarshall** [comment]
@detj can you convert your tabs to 2 spaces, to be consistent with the rest of the code?

File: src/magento.js (hunk 1 above)
Comment (detj) [inline src/magento.js]: Good idea!

**Comment by detj** [comment]
Yup, don't know how I missed it. Must be vim. Time to use Sublime :smile:

**Pull request merged**